openhub/management/commands/import_openhub_data.py

```diff
@@ -0,0 +1,23 @@
+import logging
+
+from django.core.management import BaseCommand
+from django.core.management import call_command
+
+logging.basicConfig(level=logging.DEBUG)
+
+
+class Command(BaseCommand):
+    help = 'This command invoke all the importing data command'
+
+    try:
```

Comment:
this should be in `handle`.  instantiating a Command shouldnt run it automatically.